### ISSUE TITLE
Feature Request: Add `?ref=example.com` option for external links

### ISSUE THREAD

**Issue by ElliotKillick** [Keats/gutenberg#2245]
Similar to #681, it would also be nice to have this.

Right now I'm using this `a.html` shortcode as a workaround:

`<a href="{{ l }}{% if not q %}?ref=example.com{% else %}{{ q }}&amp;ref=example.com{% endif %}{% if a %}{{ a }}{% endif %}" target="_blank">{% if t %}{{ t }}{% endif %}</a>`

`{{ a(l="https://example.com", q="?test=true", a="#test", t="Test123") }}`

Thank you for Zola, I like it a lot!

**Comment by Keats** [comment]
That's very unlikely to be added as it's not a standard HTML thing

> Thank you for Zola, I like it a lot!

Thank you!

**Comment by ElliotKillick** [comment]
Yeah, it only appears to be a common convention. I've seen some blogs (and other sites) do it for all their external links so I figured it was a builtin feature of their SSG or other blogging software.
